src/index.js.flow

```diff
@@ -0,0 +1,17 @@
+/* flow strict */
+
+declare module 'auto-check-element' {
+  declare export default class AutoCheckElement extends HTMLElement {
+    boundCheck: () => void;
+    input: HTMLInputElement;
+
+    get src(): string;
+    set src(value: string): void;
+    get csrf(): string;
+    set csrf(value: string): void;
+    get required(): boolean;
+    set required(required: boolean): void;
+
+    check(): void;
```

Comment:
Is `check` intended to be public API? When does the host app call this directly versus firing a change event at the input?